Update create-tls-listener.md

dulac opened the pull request:
*Description of changes:*
Clarified that NLBs support back-end encryption, because sometimes "renegotiation" is used in the incorrect context to mean that a LB terminates TLS and then re-encrypts (starts a new TLS session) to targets.  However, the context of "renegotiation" means something else in the TLS protocol.  Some readers could be confused, so hoping this pull request can clarify the distinction.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

dulac committed:
Update create-tls-listener.md

Clarified that NLBs support back-end encryption

koiker commented:
I would change the text to:

```
NLB's support TLS encryption between Client <-> NLB and NLB <-> Target but we do not support Client - NLB TLS renegotiation or NLB - Target TLS renegotiation.
```

Maybe point to rfc: https://tools.ietf.org/html/rfc5746

julieso committed:
Update create-tls-listener.md

Thanks for your contribution. I consulted with engineering and they asked me to remove this paragraph.
